Formula/git-gui.rb

```diff
@@ -0,0 +1,49 @@
+class GitGui < Formula
+  desc "Tcl/Tk UI for the git revision control system"
+  homepage "#{Formula["git"].homepage}"
+  url "#{Formula["git"].stable.url}"
+  sha256 "#{Formula["git"].stable.checksum}"
+  head "#{Formula["git"].head}", :shallow => false
+
+  depends_on "git"
+  depends_on "tcl-tk"
+
+  def install
+    ENV["V"] = "1" # build verbosely
+    args = %W[
+      prefix=#{prefix}
+      gitexecdir=#{libexec}/git-core
+      sysconfdir=#{etc}
+      CC=#{ENV.cc}
+      CFLAGS=#{ENV.cflags}
+      LDFLAGS=#{ENV.ldflags}
+    ]
+
+    # By setting TKFRAMEWORK to a non-existent directory we ensure that
+    # the git makefiles don't install a .app for git-gui
+    # We also tell git to use the homebrew-installed wish binary from tcl-tk.
+    # See https://github.com/Homebrew/homebrew-core/issues/36390
+    tcl_bin = Formula["tcl-tk"].opt_bin
+    args += %W[
+      TKFRAMEWORK=/we/do/not/want/to/install/a/.app/for/git-gui
+      TCL_PATH=#{tcl_bin}/tclsh
+      TCLTK_PATH=#{tcl_bin}/wish
+    ]
+    system "make", "-C", "git-gui", "install", *args
+    system "make", "-C", "gitk-git", "install", *args
+    # Git looks for git-foo binaries in $(git --exec-path), i.e.
+    # $CELLAR/git/$VERSION/libexec/git-core and in $PATH.
+    # If we only keep them in the cellar directory they will not be found.
+    # The easiest solution to this problem is to move the binaries
+    # from libexec to bin to ensure that git finds them when running e.g.
+    # `git gui` from the command line.
+    local_git_core = libexec/"git-core"
+    local_git_core.each_child do |f|
+      mv f.to_s, bin
+    end
+  end
+
+  test do
+    system libexec/"git-core/git-gui", "--version"
```

Comment:
Use `bin` here. Is there anything that can be done beyond just `--version` on the GUIs?

Reply:
Not very much since they pop up the GUI  with any other option and I'm not sure how we would test that.